src/decorators/SequentialEventListener.ts

```diff
@@ -1,21 +1,16 @@
-import type { IEventConstructor } from '../ISequentialEventListener';
+import type { IEvent } from '../ISequentialEventListener';
 import 'reflect-metadata';
+import { randomUUID } from 'crypto';
```

Comment:
random uuid was added in node 12 which may be a problem, esp because we don't specify node version in package.json

Reply:
fixed